pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "mauth-client"
-version = "1.5.1"
+version = "1.6.0"
```

Comment:
Should it be `1.5.2` instead (patch fixing broken minor version `5`)?

Reply:
went with `1.6.0` because i also decided to remove python 3.7 which is EOL tomorrow: https://endoflife.date/python

Reply:
Got it